### PR TITLE
feat(integrations): add support for braze

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe the problem and your solution
Add support for Braze

This has been added without testing, which is why the `proxy.verification` field was omitted.

### CODE REVIEW

File: docs-v2/integrations/all/braze/connect.mdx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- If you’re using the **older navigation**, you can create an API key from **Developer Console** > **API Settings**.
+3. Select **Create API Key**.
+4. Give your new key a name for identification at a glance.
+5. Specify **allowlisted IP** addresses and subnets for the new key.
+6. Select which **permissions** you want to be associated with your new key. Once you are done your api key is generated for you.
+- Keep in mind that after you create a new **API key**, you cannot edit the scope of permissions or the allowlisted IPs. If you need to change the scope of a key, create a new key with the updated permissions and implement that key in place of the old one.
+
+<img src="/integrations/all/braze/rest-api-key.png" />
+
+#### Step 2: Enter credentials in the Connect UI
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/braze/connect.mdx]:
```suggestion
#### Step 3: Enter credentials in the Connect UI
```

File: packages/shared/providers.yaml [hunk 2]
```diff
@@ -928,6 +928,36 @@ braintree-sandbox:
...
+    docs: https://docs.nango.dev/integrations/all/braze
+    docs_connect: https://docs.nango.dev/integrations/all/braze/connect
+    connection_config:
+        instanceUrl:
+            type: string
+            title: Instance URL
+            description: The REST API URL of your Braze instance
+            example: iad-02.braze.com
+            format: hostname
+            prefix: https://rest.
```
Comment (bodinsamuel) [inline packages/shared/providers.yaml:1010]: is it always braze.com? if yes, it's better to ask for a subdomain and put a `suffix: braze.com`

Comment (hassan254-prog) [reply packages/shared/providers.yaml:1010]: It can change to `.eu` for hosted customers.

File: docs-v2/integrations/all/braze/connect.mdx (hunk 1 above)
Comment (hassan254-prog) [reply docs-v2/integrations/all/braze/connect.mdx]: thanks 🙏, updated with the latest commit, also changed the case for the `retry.at` and `authorization` values

**Review by bodinsamuel — APPROVED** [review]

**Review by khaliqgant — APPROVED** [review]

**Pull request merged**